graphql_compiler/compiler/compiler_frontend.py

```diff
@@ -173,7 +173,9 @@ def _get_fields(ast):
             # If we ever allow repeated field names,
             # then we have to change the Location naming scheme to reflect the repetitions
             # and disambiguate between Recurse and Traverse visits to a Location.
-            raise GraphQLCompilationError(u'Encountered repeated field name: {}'.format(name))
+            raise GraphQLCompilationError(u'Encountered repeated field name: {}. To use '
+                                          u'multiple directives on a field, write the other '
+                                          u'directives directly below the first one.'.format(name))
```

Comment:
This advice applies only to duplicate property fields, and is very misleading for duplicate vertex fields. Can you distinguish which one it is? Also, say `after` the first one, not `below`.

Reply:
Can you actually distinguish with an if statement and separate error message? I still think this message is adding confusion when there's duplicate vertex fields.